src/components/userMenu/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import './Menu.css';
+
+class Menu extends Component {
```

Comment:
This component does not use state nor have any side effects. Do convert to a function component.

Reply:
All right.